recipes/spyder/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Normally we add these options `--single-version-externally-managed --record record.txt` if the package uses `setuptools` (or tries too). If we could try it once, we can determine whether or not it is using `setuptools` in this case.